README.md

```diff
@@ -1,140 +1,9 @@
-# Supported Windows Server 2016 Version 1709 (Fall Creators Update) amd64 tags
+dotnet-framework-docker
```

Comment:
Personally, I'm not a fan of the split READMEs. Especially given the fact that I envision a combined "How to use these images" with a multi-stage build.

Reply:
I should have called out that the plan is for two docker hub repos - `microsoft/dotnet-framework` and `microsoft/dotnet-framework-build`.  Because of this two READMEs are required - one per repo.  The patterns used here is following what is currently done in the ASP.NET Core.

I am certainly open to a discussion on the two vs one repo.